.devcontainer/devcontainer.json

```diff
@@ -5,6 +5,10 @@
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
 	"image": "mcr.microsoft.com/devcontainers/javascript-node:1-20-bullseye",
 
+	// Set workspace path for the dev container. This is the root of the project in the container.
+  	"workspaceFolder": "/workspace",
+	"workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind",
+
```

Comment:
This shouldn't be needed, what software are you using for launching devcontainers?

Reply:
I was using VS Code on Manjaro Linux